src/components/Thumbnail/Thumbnail.tsx

```diff
@@ -25,9 +26,12 @@ export function Thumbnail({source, alt, size = 'medium'}: ThumbnailProps) {
     size && styles[variationName('size', size)],
   );
 
-  return (
-    <span className={className}>
-      <Image alt={alt} source={source} className={styles.Image} />
-    </span>
-  );
+  const content =
+    typeof source === 'string' ? (
+      <Image alt={alt} source={source} />
+    ) : (
+      <Icon accessibilityLabel={alt} source={source} />
+    );
```

Comment:
Maybe I should flip this and say `source === 'function'`

Reply:
I think this is ok but should we add a couple of tests for it?

Reply:
Added some tests.